### DISCUSSION TITLE
Change Char/sec manually

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#6728] [Ideas]
Hello, an immense thank you for this piece of software!
I was wondering, could we change the char/sec manually like for the duration in this picture:
![image](https://user-images.githubusercontent.com/37704802/224274176-7982f39b-f5b1-429f-a61c-7ab051cd6c08.png)

The problem that spurred me to ask this question is that I use the auto adjust duration for 25, but for some videos they tell us to use both 25 and 30. So it works with 25 but for 30, I need to manually move the ending of the subtitle with the mouse, which is pretty tedious.

**Comment by darnn** [comment]
What do you mean by "change char/sec manually like for the duration"? What are you trying to change? Optimal chars/sec? Max chars/sec? The duration?

**Comment by devkatalist** [comment]
Do you see how you can change the duration manually? I was asking if the same thing could be done with char/sec. Of course, that would affect the duration.

**Comment by darnn** [comment]
If I understand you correctly, what you want can be accomplished by creating a different profile (in options->settings->general on the top left) and switching between the two different profiles with the different chars/sec values. But I'm still not 100% sure I actually understand if that's what you want.

**Comment by devkatalist** [comment]
That would theoretically work, but I don't think it makes sense to keep switching profiles while working on the same video just to change one simple variables (char/sec). Besides, that would defeat the purpose of having profiles.

**Comment by niksedk** [comment]
Chars/sec is not really something you can just choose as it is a combination of number of characters + duration.
You can use "Fix common errors" or "F8" + manual fix of duration... 

Do you use max chars. 25 and 30 in the same subtitle?

**Comment by devkatalist** [comment]
Yes, I have to use both in the same video. But let's suppose I could manually change it, and the characters remain unchanged, that would mean it's the duration that gets affected, right?

**Reply by niksedk** [reply]
> Yes, I have to use both in the same video. But let's suppose I could manually change it, and the characters remain unchanged, that would mean it's the duration that gets affected, right?

Yes, that's correct.

How do you know when to use max "25" and when to use max "30" chars/sec?

**Comment by devkatalist** [comment]
Depends on the context. We normally have to stick with 25, but if the speaker in the video talks a little too fast, then we are permitted exceptionally the use of 30. And by the way, I used the shortcut for "Recalculate duration (based on optimal speed)," but it doesn't work the way it's defined (in the profile, I have set max to 25 and optimal to 30).

**Comment by niksedk** [comment]
In the profile, you need to set optimal to something lower than max, e.g.  max to 25 and optimal to 17.

**Comment by devkatalist** [comment]
I switched them to be max of 30 and optimal of 25, and that solved the problem of having an alert when I used to use 30, but I still cannot have a way of choosing 30 automatically (or manually as suggested in the original post).
